src/dodal/devices/apple2_undulator.py

```diff
@@ -497,11 +491,11 @@ async def determinePhaseFromHardware(self) -> tuple[str | None, float]:
         (May be for future one can use the inverse poly to work out the energy and try to match it with the current energy
```

Comment:
Could: This is maybe a bit out of scope of the issue but could you rename this function to be `snake_case`?